watertap/unit_models/electrodialysis_0d.py

```diff
@@ -0,0 +1,938 @@
+            initialize=0.0001,
+            bounds=(1e-6, 1e-1),
+            units=pyunits.meter,
+            doc="Membrane thickness",
+        )
+        self.solute_diffusivity_membrane = Var(
+            self.membrane_set,
+            self.config.property_package.ion_set
+            | self.config.property_package.solute_set,
+            initialize=1e-10,
+            # bounds=(1e-11, 1e-6),
+            units=pyunits.meter**2 * pyunits.second**-1,
+            doc="Solute (ionic and neutral) diffusivity in the membrane phase",
+        )
+        self.ion_trans_number_membrane = Var(
+            self.membrane_set,
+            self.config.property_package.ion_set,
+            bounds=(0, 1),
+            units=pyunits.dimensionless,
+            doc="Ion transference number in the membrane phase",
+        )
+        self.water_trans_number_membrane = Var(
+            self.membrane_set,
+            initialize=5,
+            bounds=(0, 1),
+            units=pyunits.dimensionless,
+            doc="Transference number of water in membranes",
+        )
+        self.water_permeability_membrane = Var(
+            self.membrane_set,
+            initialize=5,
+            units=pyunits.meter * pyunits.second**-1 * pyunits.pascal**-1,
+            doc="Water permeability coefficient",
+        )
+        self.membrane_surface_resistence = Var(
+            self.membrane_set,
+            initialize=2e-4,
+            bounds=(1e-6, 1),
+            units=pyunits.ohm * pyunits.meter**2,
+            doc="Surface resistence of membrane",
+        )
+        self.electrodes_resistence = Var(
+            initialize=0,
+            domain=NonNegativeReals,
+            units=pyunits.ohm * pyunits.meter**2,
+            doc="areal resistence of TWO electrode compartments of a stack",
+        )
+        self.current = Var(
+            self.flowsheet().config.time,
+            initialize=1,
+            bounds=(0, 100),
+            units=pyunits.amp,
+            doc="Current across a cell-pair or stack",
+        )
+        self.voltage = Var(
+            self.flowsheet().config.time,
+            initialize=100,
+            units=pyunits.volt,
+            doc="Voltage across a stack, declared under the 'Constant Voltage' mode only",
+        )
+        self.current_utilization = Var(
+            initialize=1,
+            bounds=(0, 1),
+            units=pyunits.dimensionless,
+            doc="The current utilization including water electro-osmosis and ion diffusion",
+        )
+
+        # Performance metrics
+        self.current_efficiency = Var(
+            self.flowsheet().config.time,
+            initialize=0.9,
+            bounds=(0, 1),
+            units=pyunits.dimensionless,
+            doc="The overall current efficiency for deionizaiton",
+        )
+        self.power_electrical = Var(
+            self.flowsheet().config.time,
+            initialize=1,
+            bounds=(0, Inf),
```

Comment:
```suggestion
            bounds=(0, 1e5),
```
We should try and provide a reasonable upper limit (whatever that limit may be)